src/index.tsx

```diff
@@ -1,11 +1,16 @@
 import { createRoot } from 'react-dom/client';
 
-import './styles/index.css';
-import './styles/todo-list.css';
-import './styles/filters.css';
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
```

Comment:
Don't need to change imports 
<img width="441" alt="image" src="https://github.com/user-attachments/assets/5558d6a8-46f8-4f5f-84bf-275f32a9bc55">